core/vcpu.c

```diff
@@ -3964,6 +3964,13 @@ static int _copy_desc(segment_desc_t *old, segment_desc_t *new)
     return flags;
 }
 
+int get_vcpu_state_size( struct vcpu_t *vcpu )
```

Comment:
As for the get_vcpu_state_size(), I understand it is for forward-backward compatibility consideration for QEMU-HAXM match. It is acceptable for this patch currently. We are designing a new mechanism so that some IOCTLs are scalable in future.

Reply:
Yes, it is for forward-backward compatibility.